frontend/src/app/components/settings/settings.routing.ts

```diff
@@ -0,0 +1,11 @@
+import { Routes, RouterModule } from '@angular/router';
+import { UserSettingsComponent } from '../settings/user-settings/user-settings.component';
+import { ProjectSettingsComponent } from './project-settings/project-settings.component';
+import { SettingsComponent } from './settings.component';
+const routes: Routes = [
+    { path: 'portal/user', component: UserSettingsComponent },
+    { path: 'portal/porject', component: ProjectSettingsComponent}
+
+];
+
+export const SettingsRoutes = RouterModule.forChild(routes);
```

Comment:
Please, do routing as a SettingsRoutingModule. See example how it's done in WorkSpaceModule